package-lock.json

```diff
@@ -1,8 +1,1524 @@
 {
   "name": "azure-pipelines-tool-lib",
-  "version": "1.0.0-preview.0",
-  "lockfileVersion": 1,
+  "version": "2.0.0-preview.0",
+  "lockfileVersion": 2,
```

Comment:
Not quite sure what exactly going on here but as I can see in the master branch `version` is set to `1.3.1` why it's `1.0.0-preview.0` here? Is this branch release based on the `master` branch?

https://github.com/microsoft/azure-pipelines-tool-lib/blob/437d0e577dc2d5315798a441c6eb4bb006fdf0c7/package-lock.json#L2-L4

Also, why the version in `package.json` is already set to `2.0.0-preview.0`?

https://github.com/microsoft/azure-pipelines-tool-lib/blob/f507fdcd6d7d2f46472a951c85277cb7deb48ed4/package.json#L2-L4

Please clarify these moments